.aspect/workflows/config.yaml

```diff
@@ -5,6 +5,12 @@ bazel:
     - --enable_bzlmod
     - --remote_download_minimal
     - --nobuild_runfile_links
+    - --experimental_fetch_all_coverage_outputs
```

Comment:
best of both worlds. we can "build without the bytes" and still download the `coverage.dat` files when running with coverage enabled